detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/PackagesConfig/PackagesConfigResolver.cs

```diff
@@ -45,6 +48,8 @@ private List<NugetDependency> GetDependencies()
             System.IO.Stream stream = new System.IO.FileStream(PackagesConfigPath, System.IO.FileMode.Open, System.IO.FileAccess.Read);
             PackagesConfigReader reader = new PackagesConfigReader(stream);
             List<PackageReference> packages = reader.GetPackages().ToList();
+            
+            bool isDevDependency = ExcludedDependencyTypeUtil.isDependencyTypeExcluded(ExcludedDependencyTypes,"DEV");
```

Comment:
The variable name seems inaccurate.

Reply:
We have used the same variable name at every place for this support previously. :)

Reply:
True, and I now see that I even approved PRs where those variables were introduced in this manner.
I still vote to change it at least in this file but ideally in other files as well.

Reply:
Done